### ISSUE TITLE
[BrainHack] port the atlas BIDS spec to markdown

### ISSUE THREAD

**Issue by melanieganz** [openneuropet/outreach#32]
BrainHack Nordic 2024

Project:

Port the atlas BIDS specification from the Google doc to markdown a fork of the BIDS specification made by @peerherholz
(see BIDS spec issue [#1281](https://github.com/bids-standard/bids-specification/issues/1281))

Meetings:
Next week 24th January from 2pm PM CET -- BIDS Atlas PEB markdown stuff together [jisti meeting](https://meet.jit.si/BIDS_connectivity_project_meeting) -- see https://github.com/openneuropet/outreach/blob/main/Brainhack-Nordic2024/README.md

**Comment by CPernet** [comment]
it's already in the README

**Comment by melanieganz** [comment]
Yes, but in the spirit of showing people how to make issues and projects that are easy to find, I added this. ;-)

**Comment by melanieganz** [comment]
Anyone that wants to work on this, check out the atlas.md [here](https://github.com/PESTILLILAB/bids-specification/blob/bep038/src/atlas.md) and find me.
